homeassistant/components/ihc/translations/da.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
This file needs to be removed - no longer necessary.

Reply:
Removed